alerting/src/main/kotlin/org/opensearch/alerting/transport/TransportGetMonitorAction.kt

```diff
@@ -149,7 +149,9 @@ class TransportGetMonitorAction @Inject constructor(
                     }
 
                     override fun onFailure(t: Exception) {
-                        actionListener.onFailure(AlertingException.wrap(t))
+                        actionListener.onFailure(
```

Comment:
plz add check that exception is caused by IndexNotFoundException 
you can find such a check done here too: https://github.com/opensearch-project/security-analytics/blob/72fdaff8cc8c481eac3d0a9cb92480ed2f1a3321/src/main/java/org/opensearch/securityanalytics/threatIntel/service/DefaultTifSourceConfigLoaderService.java#L72

Reply:
plz log `t`